src/gui/optionsdialog.ui

```diff
@@ -184,29 +169,45 @@
                  </property>
                 </spacer>
                </item>
-               <item row="1" column="0">
-                <widget class="QLabel" name="label_15">
+               <item row="2" column="0" colspan="2">
+                <widget class="QCheckBox" name="checkUseSystemIcon">
                  <property name="text">
-                  <string>Theme:</string>
+                  <string>Use system icon theme</string>
                  </property>
                 </widget>
                </item>
-               <item row="1" column="1">
-                <widget class="QComboBox" name="comboTheme"/>
-               </item>
-               <item row="1" column="2">
-                <widget class="QLabel" name="lbl_i18n_info_3">
+               <item row="3" column="0" colspan="3">
+                <widget class="QLabel" name="label_15">
                  <property name="font">
                   <font>
                    <italic>true</italic>
                   </font>
                  </property>
                  <property name="text">
-                  <string>(Requires restart)</string>
+                  <string>Note: Changes Requires Restart</string>
```

Comment:
Why change it? Anyway if you decide to proceed with this then Requires->Require.

Reply:
I had changed it. can you take a look again?